_posts/2018-03-07-bug-bbq.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+permalink: 2018-03-07-bug-bbq
+subheadline: "Bug BBQ"
+title: "Announcing the Spring 2018 Bug BBQ"
+teaser: "Mark your calendar, the Spring 2018 Bug BBQ will take place April 12-13!"
+header:
+    image_fullwidth: "light-blue-wood-texture.jpg"
+categories:
+    - blog
+comments: true
+show_meta: true
+authors: ["François Michonneau", "Erin Becker"]
+---
+
+
+The Carpentries are preparing to publish the **Social Sciences** and the **Geospatial** curricula on [Zenodo](https://zenodo.org/) by mid-May. This will be the first 
+publication for these lessons. We regularly publish our lessons ([SWC](https://zenodo.org/communities/swcarpentry/), [DC](https://zenodo.org/communities/datacarpentry/)) 
+to provide stable identifiers for polished versions of
+the lessons. This enables referenced discussions of the lesson materials and gives contributors a verifiable product to cite on their CVs 
+or resumes.
+
+This release will include the following lessons:
+
+##### Geospatial Curriculum
+
+* [Introduction to Geospatial analysis](https://github.com/datacarpentry/r-spatial-data-management-intro)
+* [Geospatial analysis in R](https://github.com/datacarpentry/R-spatial-raster-vector-lesson)
+
+##### Social Science Curriculum
+
+* [Spreadsheet organization](https://github.com/datacarpentry/spreadsheets-socialsci)
+* [SQL](https://github.com/datacarpentry/sql-socialsci)
+* [Python](https://github.com/datacarpentry/python-socialsci)
+* [OpenRefine](https://github.com/datacarpentry/openrefine-socialsci)
+* [R](https://github.com/datacarpentry/r-socialsci)
+
+
+## Get involved!
+
+If you've made a contribution to one of these lessons, you're already an author. Help make sure the final product is polished and complete by getting involved in the lesson release.
+
+We are organizing a Bug BBQ to prepare these lessons for release. The main goal for the Bug BBQ is to get the Geospatial and Social Science lessons ready for release. However, if you are a Maintainer for another lesson, and you are available and interested in getting some extra eyes on your lessons, let [François](mailto:francois@carpentries.org) know and we'll include your repository on the Bug BBQ website.
```

Comment:
I like this language.

Reply:
Can we also follow up with specific Maintainers and ask if they want to be included. I'm thinking of:

- DC Genomics R (not published)
- DC Genomics Project organization and management
- SWC Plotting and Programming in Python (not published?)

Reply:
yes, I'll follow up with them individually tomorrow.